proposals/0000-template.md

```diff
@@ -81,6 +81,24 @@ Hopefully this section will be empty by the time the proposal is brought to
 the steering committee.
 
 
+## Proposal Collaboration
+
+Choose **one** of the following two statements (and delete the other), or write
+your own that addresses this theme:
+
+1. Writing precise specifications is hard. Accordingly, I invite the committee
+   members to collaborate in writing this proposal, helping to make the specification
+   of my new features as tight as possible and easiest to implement. These
+   external contributions are meant to clarify my intent, not change it, and I retain
+   the ability to shape the final proposal.
+   
+2. Writing precise specifications is hard. Accordingly, I seek to increase my
+   ability to perform this skill and welcome constructive, detailed criticism
+   in how to make my specification more precise. I recognize that doing so may
+   take many rounds and will interpret others' advice as a way to improve my
+   craft. If I get frustrated with the process, I will speak up early.
+
+
```

Comment:
Something about the way this is set up feels odd; it seems we are putting rather personal, almost emotional wording into other people’s mouths.

And do we really want such boiler plate code in all proposals, even after acceptance?

Maybe the shepherd should check if the proposal even needs help in getting more precise, and if so, use the GitHub thread or private communication to see what kind of help the author needs and wants?

Reply:
Perhaps the wording can be improved. I agree that it's odd to ascribe emotions to others. And I agree that keeping the boilerplate in after acceptance is poor. Perhaps we remove it then, or perhaps this communication is made on another frequency.

But I do want the communication. This is an attempt to resolve a fundamental tension in our proposals process: we want to be inclusive of, say, people without PhDs in PL theory; and we want very precise specifications that can last in perpetuity. And individuals differ in how they want to bridge this tension: some would prefer a proposal to be their own work, no matter how many times it has to be revised; and some would prefer a person more experienced in this kind of thing to just fix it up. If we don't know where an individual sits on this spectrum, we'll offend some people (and discourage future proposals from them). And it can be awkward to ask this question in the middle of technical debate. So I just wanted it up front.

I'm very happy for other mechanisms to achieve this goal -- or of course I could be making much out of little here.

Reply:
Perhaps instead of putting this boilerplate language in the template, we explain in the README that drafting the proposal should be a collaborative process, that the committee understands that not everyone has the background to specify a change with the precision the committee desires, and that we're happy to help with the actual drafting in addition to providing feedback.

I think the point of putting this language in the README instead of the template is that it sets a general expectation, rather than putting words in the author's mouth.